frontend/src/components/GrantAccess.vue

```diff
@@ -94,6 +101,27 @@ function toRequestPayload(formData: any) {
 function statusSelected(evt: any) {
     formData.value.role = JSON.parse(evt.target.value);
 }
+
+async function getForestClientNumber(evt: string) {
```

Comment:
I would suggest to change the parameter "evt" to a better name, like call it "forestClientNumber", when we name the parameter, it's better to give a name that make more sense, like in this case, we know this function will take a string.

Reply:
Thanks, it will be changed as suggested.